source/Animate.h

```diff
@@ -113,6 +115,14 @@ Animate<T>::operator const T &() const
 
 
 
+template <typename T>
+bool Animate<T>::IsAnimationDone() const
+{
+	return steps == 0;
```

Comment:
!steps would work as well but its fine as is too

Reply:
I try not to use boolean `!` for non-boolean variables. Since `steps` is a counter, I figured this would be more clear.